patch/kernel/archive/meson64-6.11/dt/meson-axg-amper-gateway-am-gz80x.dts

```diff
@@ -20,49 +18,15 @@
 		ethernet0 = &ethmac;
 	};
 
-	chosen {
-		stdout-path = "serial0:115200n8";
-	};
-
-	leds {
-		compatible = "gpio-leds";
-
-		led-blue {
-			color = <LED_COLOR_ID_BLUE>;
-			function = LED_FUNCTION_STATUS;
-			gpios = <&gpio_ao GPIOAO_3 GPIO_ACTIVE_HIGH>;
-			linux,default-trigger = "default-on";
-		};
-
-		led-green {
-			color = <LED_COLOR_ID_GREEN>;
-			function = LED_FUNCTION_STATUS;
-			gpios = <&gpio_ao GPIOAO_4 GPIO_ACTIVE_HIGH>;
-			linux,default-trigger = "mmc1";
-		};
-
-		led-red {
-			color = <LED_COLOR_ID_RED>;
-			function = LED_FUNCTION_STATUS;
-			gpios = <&gpio_ao GPIOAO_7 GPIO_ACTIVE_HIGH>;
-			linux,default-trigger = "usb-host";
-		};
- 	};
-
 	/* 1024MB RAM */
 	memory@0 {
 		device_type = "memory";
 		reg = <0x0 0x0 0x0 0x40000000>;
 	};
 };
 
-/delete-node/ &i2c1;
-
-/* wifi module */
 &sd_emmc_b {
-	non-removable;
-
-	rtl8189ftv: wifi@1 {
+	sdio: wifi@1 {
```

Comment:
do you really need this lines? kernel can autodiscovery wifi modules.

Reply:
I don't believe so. But it seems to be common practice to put it in there. So I just followed the example of others.

Reply:
One example: https://github.com/torvalds/linux/blob/master/arch/arm64/boot/dts/amlogic/meson-g12b-bananapi-cm4.dtsi#L313

It doesn't really serve any purpose from what I can tell other than to say "this is that". The schematic suggests a diff realtek wifi module would be there, so I thought lets just name it something generic.

Reply:
I still couldn't figure out the subtle cases of ideologically adding parameters. Anyway, it doesn't change anything.